server/db/models/film.js

```diff
@@ -12,6 +12,7 @@ var filmSchema = new mongoose.Schema({
   price: {type: Number, required: true},
   photo: {type: String, default: 'http://placehold.it/200x300'},
   inventory: {type: Number, required: true},
+  purchased: {type: Number, required: true}
```

Comment:
We can. Doesn't need to be a required field or present in the seed to work though. Will track purchases from app and add stats over time.